filesystems/oss/src/main/scala/cromwell/filesystems/oss/nio/OssFileChannel.scala

```diff
@@ -0,0 +1,19 @@
+package cromwell.filesystems.oss.nio
```

Comment:
@crisish Absolutely not a blocker for merging this but you all will want to consider splitting the nio implementation out into a separate library. That way GATK4 can make use of it as well

Reply:
Ok, I'll try to update this NIO interface to our OSS team.

Reply:
@crisish Cool. Like I said, no need to do it before merging the PR, can always be done later